### DISCUSSION TITLE
Inav 7

### DISCUSSION THREAD

**Discussion by Fabwize** [iNavFlight/inav#9203] [Q&A]
Hello, can I have inav 7 target to test the Vtol conversion on NanoDrak?
i ave F411 TWE.Matek
 THANKS

**Comment by stronnag** [comment]
No, from the [documentation](https://github.com/shota3527/inav/blob/sh_mixer_profile/docs/MixerProfile.md), you would see it is not available on F411, F722 due to lack of resources (e.g. flash size).

**Reply by Fabwize** [reply]
thank you, I also have a H743 light Matek

**Comment by MaiaNunes** [comment]
Wow. Just heard that INAV 7 is coming with VTOL capabilities.
Is it possible to get the target for my new SpeedyBee F405 Wing ?
Thank you

**Reply by DzikuVx** [reply]
it's kind of too early for that I think

**Comment by luciduous** [comment]
It's been a minute guys since I've been in the trenches. Last time I was active, I was using INAV 4.0. Trying to get my head wrapped around all the improvements since then, but great to hear 7.0 is touting VTOL capabilities. My own inhouse designed penta-copter utilizes a center large prop (CCW) with the remaing 4 outer small props in standard quad format (CW). The center prop transitions for the forward flight. Originally got this system working with a KK2 board, and finally now getting my Matek h743 slim outfitted with the system. Can't wait to jump into the deep end of 7.0 to get this bird up and airborn! It's a big drone too (~3' across) so this will be fun. 
![image](https://github.com/iNavFlight/inav/assets/91760616/82a5c42a-f0e3-41ec-93c2-a98ff744a3ee)
